pkg/cmd/util/clientcmd/factory_client_access.go

```diff
@@ -264,16 +263,8 @@ func (f *ring0Factory) SuggestedPodTemplateResources() []schema.GroupResource {
 	return f.kubeClientAccessFactory.SuggestedPodTemplateResources()
 }
 
-// Saves current resource name (or alias if any) in PrintOptions. Once saved, it will not be overwritten by the
-// kubernetes resource alias look-up, as it will notice a non-empty value in `options.Kind`
 func (f *ring0Factory) Printer(mapping *meta.RESTMapping, options kprinters.PrintOptions) (kprinters.ResourcePrinter, error) {
-	if mapping != nil {
-		options.Kind = mapping.Resource
-		if alias, ok := resourceShortFormFor(mapping.Resource); ok {
-			options.Kind = alias
-		}
-	}
-	return describe.NewHumanReadablePrinter(f.JSONEncoder(), f.Decoder(true), options), nil
```

Comment:
ah, I see.  Apparently we have a different printer here because why not have a different printer here that is only sometimes used.